2200041/14654.py

```diff
@@ -0,0 +1,14 @@
+#14654
+T = int(input())
+
+for t in range(1, T+1):
+    card = input()      # len을 위해 str로 받았다.
+    if "-" in card:     # -를 뺀다
+        card = card.replace("-", "")
+    if len(card) == 16:
+        if int(card[0:1]) in [3, 4, 5, 6, 9]:
```

Comment:
깔끔한 풀이네요 👍 
```suggestion
        if int(card[0]) in [3, 4, 5, 6, 9]:
```
0부터 1까지 인덱스는 0번 인덱스 하나와 똑같아요!